src/gui/accountsettings.cpp

```diff
@@ -1110,10 +1110,7 @@ void AccountSettings::slotAccountStateChanged()
                                        Utility::escape(safeUrl.toString()));
         QString serverWithUser = server;
         if (AbstractCredentials *cred = account->credentials()) {
-            QString user = account->davDisplayName();
-            if (user.isEmpty()) {
-                user = cred->user();
-            }
+            QString user = account->prettyName();
```

Comment:
`const auto`

Reply:
Done, though again I did more than just this